x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/optimizer/OptimizerTests.java

```diff
@@ -152,6 +152,27 @@ public void testEqualsWildcardOnRight() {
         }
     }
 
+    public void testEqualsWildcardQuestionmarkOnRight() {
+        List<String> tests = Arrays.asList(
```

Comment:
Why not defining a list when it's actually needed? Or add one more query. It's a bit too much to use a list for just one query imo. I remember there is one more test like this for `*` (using a list for one query only).